src/Microsoft.IdentityModel.Logging/ConfigurationManagerTelemetryInstrumentation.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+
+namespace Microsoft.IdentityModel.Logging
+{
+    internal class ConfigurationManagerTelemetryInstrumentation : ITelemetryInstrumentation
+    {
+        public void IncrementOperationCounter(string clientVer, string operationStatus)
+        {
+            var tagList = new TagList()
+            {
+                { TelemetryConstants.IdentityModelVersionTag, IdentityModelTelemetryUtil.ClientVer },
+                { TelemetryConstants.OperationStatusTag, operationStatus }
+            };
+
+            IdentityModelTelemetry.IncrementConfigurationManagerCounter(tagList);
+        }
+
+        public void IncrementOperationCounter(string clientVer, string operationStatus, string exceptionType)
+        {
+            var tagList = new TagList()
+            {
+                { TelemetryConstants.IdentityModelVersionTag, IdentityModelTelemetryUtil.ClientVer },
+                { TelemetryConstants.OperationStatusTag, operationStatus },
+                { TelemetryConstants.ExceptionTypeTag, exceptionType }
+            };
+
+            IdentityModelTelemetry.IncrementConfigurationManagerCounter(tagList);
+        }
+
+        public void LogOperationDuration(string clientVer, long durationInMilliseconds)
+        {
+            var tagList = new TagList()
```

Comment:
This one can be static

Reply:
My understanding is that it cannot implement the interface member if it's static